src/com/seafile/seadroid2/ui/activity/ShibbolethAuthorizeActivity.java

```diff
@@ -109,14 +151,14 @@ public void onReceivedSslError(WebView view, SslErrorHandler handler, SslError e
 
         @Override
         public void onPageFinished(WebView webView, String url) {
-            Log.d(DEBUG_TAG, "onPageFinished " + url);
+            Log.d(DEBUG_TAG, "onPageFinished " + serverUrl);
             showPageLoading(false);
 
-            String cookie = getCookie(url, SEAHUB_SHIB_COOKIE_NAME);
+            String cookie = getCookie(serverUrl, SEAHUB_SHIB_COOKIE_NAME);
             if (cookie == null)
                 return;
 
-            Account account = parseAccount(url, cookie);
+            Account account = parseAccount(serverUrl, cookie);
```

Comment:
The server url must be ended with a backword slash (`/`). Here we need similar things like `cleanServerURL` in https://github.com/haiwen/seadroid/tree/tmp/src/com/seafile/seadroid2/ui/activity/AccountDetailActivity.java#L215